### ISSUE TITLE
AttributeError: 'oead.byml.Array' object has no attribute 'index'

### ISSUE THREAD

**Issue by ArchLeaders** [NiceneNerd/BCML#322]
This error was reported by a user on Discord when merging mods. They _fixed_ the issue by downgrading to BCML `3.8.4`

The mods in question:
![image](https://user-images.githubusercontent.com/80713508/166605832-df5f5604-6d66-4b19-a6d6-5f99a6198d57.png)

I attempted to re-create the error using the same mods (in the same order), but with no success. I remerged with the files in `3.8.5`. Upon doing so, the operation failed, throwing `AttributeError: 'oead.byml.Hash' object has no attribute 'pop'` as expected.

The second attempt was with the files of the latest commit on GitHub. When remerging with these files, the operation was completed successfully.

I will leave this issue here in case someone ~~ginger~~ knows more about it than me.

```
Traceback (most recent call last):
  File "e:\python\lib\site-packages\bcml_api.py", line 550, in remerge
    install.refresh_merges()
  File "e:\python\lib\site-packages\bcml\install.py", line 661, in refresh_merges
    merger.perform_merge()
  File "e:\python\lib\site-packages\bcml\util.py", line 462, in timed_function
    res = func(*args, **kwargs)
  File "e:\python\lib\site-packages\bcml\mergers\quests.py", line 164, in perform_merge
    quest_index = 0 if add["prev_quest"] == "--index_zero" else quests.index(add["prev_quest"]) + 1
AttributeError: 'oead.byml.Array' object has no attribute 'index'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "e:\python\lib\site-packages\bcml_api.py", line 33, in status_run
    data = func(*args, **kwargs)
  File "e:\python\lib\site-packages\bcml\install.py", line 324, in do_and_refresh
    res = func(*args, **kwargs)
  File "e:\python\lib\site-packages\bcml_api.py", line 558, in remerge
    raise Exception(
Exception: There was an error merging your mods. 'oead.byml.Array' object has no attribute 'index'
Note that this could leave your game in an unplayable state.
```

**Comment by ArchLeaders** [comment]
One thing I failed to notice earlier, I do not have an exact replica of the user's mods. I downloaded the latest version of Second Wind from GitHub (1.9.12<ins>a</ins>). The user is using 1.9.12<ins>b</ins>.

**Comment by GingerAvalanche** [comment]
Fixed in v3.8.6
